tests/test_pipeline/test_schedule/test_zerobubble_pp.py

```diff
@@ -876,14 +870,14 @@ def run_with_booster_moehybridplugin(config: Tuple[int, ...]):
                 return_outputs=True,
             )
             # stage 0 chunk 0
-            parallel_output = None
             if (
                 booster.plugin.stage_manager.is_first_stage(ignore_chunk=True)
                 and rank == dist.get_process_group_ranks(plugin.pp_group)[0]
             ):
                 parallel_output = sharded_output["loss"]
             else:
                 parallel_output = torch.tensor(12345.0, device="cuda")
+            print(f"rank {dist.get_rank()} parallel_output {parallel_output}")
```

Comment:
remove debug info

Reply:
> remove debug info

Fixed in https://github.com/hpcaitech/ColossalAI/pull/6114/commits/dafda0fb7082506ad76b5deff3024b3d5dbb904b